airflow/models.py

```diff
@@ -1685,7 +1691,7 @@ def dry_run(self):
 
     @provide_session
     def handle_failure(self, error, test_mode=False, context=None, session=None):
-        self.log.exception(error)
+        self.log.error(str(error))
```

Comment:
This looks incorrect - log.exception basically does this already (but includes useful traceback). Is there some reason we need this change?

Reply:
Oh were we actually logging the error twice? I think this line should stay (unmodified) and the log at the end just removed.

Reply:
sorry, I somehow thought that `error` is just a string (prolly it was when I first wrote it), and as we are passing a string error message, was seeing `NoneType: None` at the end of the log. Will revert it back. And yes, we were printing the same message twice so removed the last one.